card/lib/card/cache.rb

```diff
@@ -25,7 +25,7 @@ class Cache
     class << self
       def [] klass
         raise "nil klass" if klass.nil?
-        cache_by_class[klass] ||= new :class=>klass, :store=>(@@using_rails_cache ? nil : Rails.cache)
+        cache_by_class[klass] ||= new :class=>klass, :store=>(@@using_rails_cache ? nil : Cardio.cache)
```

Comment:
I changed it to rails because what was there before broke in the browser.  If this works, it's better.

Even with our tests, it's probably worth a browser sanity check for any significant amount of code change.  (Especially something like this, which explicitly does not get run in test environments!)